etna/search/views.py

```diff
@@ -294,10 +294,10 @@ def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
         data = super().get_datalayer_data(request)
         if self.form.cleaned_data.get("group"):
             custom_dimension8 = (
-                self.title_base + ": " + self.form.cleaned_data.get("group")
+                self.search_tab + ": " + self.form.cleaned_data.get("group")
             )
         else:
-            custom_dimension8 = self.title_base + ": " + "none"
+            custom_dimension8 = self.search_tab + ": " + "none"
```

Comment:
Why  custom_dimension8 is a tuple here and the else is not?

Reply:
> Why custom_dimension8 is a tuple here and the else is not?

custom_dimension8  is not a tuple, its a string enclosed in brackets. **Probably from some changes earlier. Not with this code base.**